eve/main.yml

```
@@ -88,6 +93,12 @@ stages:
           env:
             <<: *deploy-env
             <<: *keycloak-env
+      - ShellCommand:
+          name: upload test coverage
```

I wonder if coverage upload should "always run" even if the previous step have failed. I'm starting to think that we only want to upload successful results. But I could be missing something.

If we want to always run, I suggest we add the option

```suggestion
          name: upload test coverage
          alwaysRun: true
```
If not then ignore my comment.

---

I would say no. The reason being, depending on which part of the build failed could result in incorrect coverage. An example is if we have a syntax error in some test that fails to run, we might miss out on otherwise tested code. Or if a build fails then the coverage is meaningless anyway.